packages/blade/src/components/ListView/_decisions/decisions.md

```diff
@@ -0,0 +1,481 @@
+# List View Pattern API Decision
+
+> [!NOTE]
+>
+> Questions like what is pattern, why are we building these patterns, and scope of pattern are answered in [pattern-terminology.md](./pattern-terminology.md)
+
+List View Pattern is a UI pattern that includes Table, Filters, and Search of the table. It is a commonly used pattern on Razorpay dashboards to visualize data and help consumers filter / search through that data.
+
+![alt text](image-5.png)
+
+## API
+
+### With Layout Component
+
+```jsx
+<ListView>
+  <ListViewFilters
+    quickFilters={[
+      {
+        title: 'All',
+        onClick: () => {},
+      },
+      {
+        title: 'Pending',
+        onClick: () => {},
+      },
+    ]}
+    onSearchChange={() => {
+      /* filter on search */
+    }}
+  >
+    <FilterChipGroup>
+      <Dropdown>
+        <FilterChip />
+        <DropdownOverlay></DropdownOverlay>
+      </Dropdown>
+
+      <Menu>
+        <FilterChip />
+        <MenuOverlay>
+          <TextInput />
+        </MenuOverlay>
+      </Menu>
+    </FilterChipGroup>
+  </ListViewFilters>
+  <Table data={{ nodes: filteredNodes }} />
+</ListView>
+```
+
+**Pros:**
+
+- Layout can be controlled on design-system end which means in future if we decide to change layout of the list view pattern, consumers will only have to upgrade version
+- Responsive layout can also be defined on design-system layer which other people might miss
+- Consistent layout across all pages
+- Functionality such as hide / show filters, placement of search, etc comes out of the box
+- Less verbose compared to API without layout component
+
+**Cons:**
+
```

Comment:
Is it really a con? Lets check with design if this will ever be needed to move things around within quick filters

Reply:
Yup it might come in as we discussed earlier on call. Turning 2 individual inputs to InputGroup, moving items to other side, etc might come in future